test/e2e/framework/ingress_utils.go

```diff
@@ -297,20 +316,20 @@ func createSecret(kubeClient clientset.Interface, ing *extensions.Ingress) (host
 	var s *v1.Secret
 	if s, err = kubeClient.Core().Secrets(ing.Namespace).Get(tls.SecretName, metav1.GetOptions{}); err == nil {
 		// TODO: Retry the update. We don't really expect anything to conflict though.
-		framework.Logf("Updating secret %v in ns %v with hosts %v for ingress %v", secret.Name, secret.Namespace, host, ing.Name)
+		Logf("Updating secret %v in ns %v with hosts %v for ingress %v", secret.Name, secret.Namespace, host, ing.Name)
 		s.Data = secret.Data
 		_, err = kubeClient.Core().Secrets(ing.Namespace).Update(s)
 	} else {
-		framework.Logf("Creating secret %v in ns %v with hosts %v for ingress %v", secret.Name, secret.Namespace, host, ing.Name)
+		Logf("Creating secret %v in ns %v with hosts %v for ingress %v", secret.Name, secret.Namespace, host, ing.Name)
 		_, err = kubeClient.Core().Secrets(ing.Namespace).Create(secret)
 	}
 	return host, cert, key, err
 }
 
-func cleanupGCE(gceController *GCEIngressController) {
-	pollErr := wait.Poll(5*time.Second, framework.LoadBalancerCleanupTimeout, func() (bool, error) {
+func CleanupGCE(gceController *GCEIngressController) {
```

Comment:
Since this is moving to the framework package, it probably needs a more descriptive name. Like CleanupGCEIngressController.